src/Cache/CacheEngine.php

```diff
@@ -178,8 +178,6 @@ public function setMultiple($values, $ttl = null): bool
                 $this->setConfig('duration', $restore);
             }
         }
-
-        return false;
```

Comment:
How come?

Reply:
Because the `try` block has return statement, so if there's no exception method will return and if there is exception then too this statement will never be reached.

I too was confused earlier and opened an issue with phpstan but later understood.

Reply:
https://3v4l.org/rCD4o
https://3v4l.org/MJ5VY